src/SFA.DAS.LevyTransferMatching.Web/Views/Pledges/Application.cshtml

```diff
@@ -88,8 +88,7 @@
                 </span>
             </summary>
             <div class="govuk-details__text">
-                <p>The estimated total cost of training is based on the funding band maximum.</p>
-                <p>The estimated cost of training for this financial year is based on the funding band maximum, the planned start date of training and the 20% completion payment.</p>
+                <p>The estimated total cost of training is based on the funding band maximum, the planned start date of training and the 20% completion payment.</p>
```

Comment:
Looking at the screenshot in the ticket, I think the styling on this needs to be changed. Looks like it should be a subheading rather than an expandable text box now.